frontend/packages/ux-editor/src/components/FormComponent.module.css

```diff
@@ -4,6 +4,7 @@
   margin-left: var(--margin-left-with-handle);
   margin-right: var(--margin-right-with-buttons);
   min-height: 80px;
+  cursor: pointer;
```

Comment:
We should probably not add `cursor: pointer` here. We do not want to tell the users that our wrapper is clickable. The wrapper it's also a non-semantic HTML-tag (div) with an onClick event
```suggestion
```

Reply:
clarification, this should probably be added on the form-component instead. :)